jwst/scripts/asn_edit.py

```diff
@@ -1,63 +1,54 @@
 #!/usr/bin/env python
 
+"""Parse command line, read, edit, write association file."""
```

Comment:
This is unclear; what does "parse command line" mean?  Suggest something like "Edit an association file from the command line."